### PR TITLE
MyRocks doc updates for next release

### PR DESCRIPTION
**Pull request by azhebel**

### CODE REVIEW

File: doc/source/myrocks/variables.rst [hunk 1]
```diff
@@ -831,6 +863,19 @@ by :variable:`rocksdb_compaction_sequential_deletes`.
 Default value is ``0``.
 Allowed range is up to ``2000000`` (two million).
 
+.. variable:: rocksdb_concurrent_prepare
+
+  :version 5.7.20-18: Implemented
```
Comment (george-lorch) [inline doc/source/myrocks/variables.rst:868]: Missing :cli: --rocksdb-concurrent-prepare ?

File: doc/source/myrocks/variables.rst [hunk 2]
```diff
@@ -1259,32 +1359,31 @@ Allowed range is up to ``18446744073709551615``.
 .. note:: A value of ``4194304`` (4 MB) is reasonable
    to reduce random I/O on XFS.
 
-.. variable:: rocksdb_max_background_compactions
+.. variable:: rocksdb_manual_wal_flush
 
-  :version 5.7.19-17: Implemented
-  :cli: ``--rocksdb-max-background-compactions``
+  :version 5.7.20-18: ``--rocksdb-manual-wal-flush``
```
Comment (george-lorch) [inline doc/source/myrocks/variables.rst]: Missing :cli: --rocksdb-manual-wal-flush ?

**Comment by george-lorch** [comment]
Ping @hrvojem

**Comment by patrickbirch** [comment]
Variables have been added

**Pull request closed without merging**